cmd/webhook/main.go

```diff
@@ -14,16 +21,50 @@ const (
 var (
 	listenAddress = flag.String("listen", "0.0.0.0", "listen address")
 	listenPort    = flag.String("port", "5000", "port to listen on")
+
+	testHooks = flag.Bool("testhooks", false, "Test webhook URI uniqueness and quit?")
+
+	useTLS  = flag.Bool("tls", false, "Use TLS? Must specify -tlskey, -tlscert, -cacert")
+	tlsKey  = flag.String("tlskey", "", "TLS Key for TLS")
+	tlsCert = flag.String("tlscert", "", "TLS Certificate")
+	caCert  = flag.String("cacert", "", "CA Cert file")
```

Comment:
The descriptions for all of these flags should be more `descriptive` of what they are used for.

Reply:
fixed